magpie/utils.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import division
-
-import cPickle as pickle
+try:
+    import cPickle as pickle
+except ImportError:
+    import _pickle as pickle
```

Comment:
I think `pickle` uses `_pickle` underneath by default, so only `import pickle` in the except clause would be enough